content/en/blog/_posts/2022-12-26-pod-scheduling-readiness.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Pod Scheduling Readiness to Alpha"
```

Comment:
```suggestion
title: "Kubernetes 1.26: Pod Scheduling Readiness"
```
Features don't really graduate to alpha, as alpha is the first stage we tell end users about.